tasks/libs/common/check_tools_version.py

```
@@ -102,3 +103,14 @@ def check_tools_version(ctx: Context, tools_list: list[str], debug: bool = False
     if should_exit:
         raise Exit(code=1)
     return True
+
+
+def check_tools_installed(tools: list) -> bool:
+    """
+    Check if the tools are installed in the system.
+    """
+    for tool in tools:
+        if not shutil.which(tool):
+            print(f"{tool} is not installed. Please install it before running this task.")
```

With this message I think we prefer to throw an exception here. You could also use `sys.stderr` / `color_message` / `Error/Warning` prefix to make this more clear. Otherwise, why not adding `Info` suffix?

NIT: We can throw an exception after the loop such that we have one message with all missing tools

---

The exception is thrown by the caller (with a raise Exit), so I think we're fine.